### DISCUSSION TITLE
Send to many - Bisq wallet

### DISCUSSION THREAD

**Discussion by TinF0ilHat1** [bisq-network/bisq#6616] [Ideas]
Hi guys , 

do you think it could be worth to add  "send to many (addresses)" option when spending from Bisq wallet?

i think it might slightly increase entropy on the blockchain.

**Comment by w0000000t** [comment]
Trying to consider this with a developer's mindset, I could tell you that, pending the additional expense for one more tx, you can withdraw to an external address first, and then send to many from there, so developer time can be more productively spent on higher priority tasks

**Reply by TinF0ilHat1** [reply]
Fair

**Comment by apemithrandir** [comment]
An advanced user could also open the Bisq wallet within Sparrow wallet and take advantage of the privacy features within that software. Though extra care would need to be taken if you have any active offers or open trades but I think @w0000000t has the correct sentiment.

If you fund from Bisq Wallet when initiating transactions then I would recommend sweeping the Bisq Wallet to a Sparrow Wallet every once in a while and start up a new Bisq Wallet to avoid the linking of transactions over a long period of time.

**Reply by TinF0ilHat1** [reply]
> An advanced user could also open the Bisq wallet within Sparrow wallet and take advantage of the privacy features within that software. 

Right, i didn't take into consideration that option.
